fsharp-backend/tests/Tests/BwdServer.HttpBytes.Tests.fs

```diff
@@ -0,0 +1,361 @@
+/// Tests the builtwithdark.com server (`BwdServer`),
```

Comment:
Can having a separate test server be avoided? I don't mind the logic is a separate file or module if it's different. But we're going to have to maintain both of these long term (and more in the future)

Reply:
(there's still tidying up to do, but) done!